contracts/HubPoolInterface.sol

```diff
@@ -23,9 +23,9 @@ interface HubPoolInterface {
         // deposits in this bundle. If non-zero then it must be set on the SpokePool's RelayerRefundLeaf amountToReturn
         // as -1 * this value to indicate if funds are being sent from or to the SpokePool.
         int256[] netSendAmounts;
-        // This is only here to be emitted in an event to track a running unpaid balance between the L2 pool and the L1 pool.
```

Comment:
nit: additional changes:
- Line 17: Remove "This array is grouped with the two above"
- Line 39: Replace "The following arrays" with "The bundleLpFees, netSendAmounts, and runningBalances"

Reply:
o sorry. 17 was a result of a merge issue. this was taken out before but came back.
doing on 39.